net/pdnsd/files/pdnsd.init

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2019 OpenWrt.org
+
+USE_PROCD=1
+
+START=65
+STOP=10
+
+PROG=/usr/sbin/pdnsd
+USER=root
```

Comment:
There's a run_as parameter but here it's being set to root. It's not necessary to set root as it is already default.